docs/UserGuide.md

```diff
@@ -211,18 +211,45 @@ For first-time users of the application, it is highly recommended for you to rea
 
 #### 3.2.1 User Interface Overview:
 <a id="user-interface-overview"></a>
-TO BE DONE 
-## To update UI component picture here:
-![startingGUI.png](images/startingGUI.png)
 
-### Add UI Components descriptions here
+![UserInterfaceOverview.png](images/UserInterfaceOverview.png)
+
+The designated names for each component of the User Interface will be used consistently throughout the User Guide to ensure clarity and minimize confusion. Below is a detailed description of each component.
+
+* **Menu Box:** Positioned at the top of the interface, the menu bar offers access to a variety of functions and features within TrackMate.
+
+* **Logo:** This represents the logo of our application, TrackMate.
+
+* **Command Box:** This designated area is where users can input commands for TrackMate to execute.
+
+* **Result Box:** Prominently displayed on the interface, this box shows the results of executed commands.
+
+* **Student List Panel:** This panel visually represents the roster of students stored within TrackMate.
+
+* **Student Panel Card:** Within the Student List Panel, each student is depicted in a concise overview displayed in card format.
+
+* **Student Card:** To view detailed information about an individual student, users can refer to the Student Card, which provides comprehensive details stored in the system.
 
 #### 3.2.2 Additional UI Components:
 
 <a id="additional-ui-components"></a>
-![startingGUI.png](images/startingGUI.png)
+![additionalUIComponents.png](images/additionalUIComponents.png)
```

Comment:
Good to see that you have updated the latest UI. However, is the logo of the application updated as well?